lib/src/pubspec.dart

```diff
@@ -388,6 +388,43 @@ class Pubspec {
   bool _parsedPublishTo = false;
   String _publishTo;
 
+  /// The list of patterns covering _false-positive secrets_ in the package.
+  ///
+  /// This is a list of git-ignore style patterns for files that should be
+  /// ignored when trying to detect possible leaks of secrets during
+  /// package publication.
+  List<String> get falseSecrets {
+    if (_falseSecrets == null) {
+      final falseSecrets = <String>[];
+
+      // Throws a [PubspecException]
+      void _falseSecretsError(SourceSpan span) => _error(
+            '"false_secrets" field must be a list of git-ignore style patterns',
```

Comment:
Another weakness of having these in the pubspec, is that they cannot be placed in sub-directories....